terraform/globals.tf

```diff
@@ -57,13 +57,13 @@ variable "vagrant_cidr" {
 }
 
 variable "dns_zone_id" {
-  description = "Amazon Route53 DNS zone identifier"
-  default     = "Z3SI0PSH6KKVH4"
+  description = "Amazon Route53 DNS zone identifier. Different per account."
+  default     = "dummy_default"
 }
 
 variable "dns_zone_name" {
-  description = "Amazon Route53 DNS zone name"
-  default     = "cf.paas.alphagov.co.uk"
+  description = "Amazon Route53 DNS zone name. Differs per account."
+  default     = "dummy_default"
```

Comment:
Yes. But you would still need to add defaults. So to avoid duplication, we moved them to globals. There are other variables in globals that are not used by _all_ our TF jobs.